Jenkinsfile

```diff
@@ -32,6 +33,7 @@ pipeline {
     booleanParam(name: 'test_ci', defaultValue: false, description: 'Enable test')
     booleanParam(name: 'release_ci', defaultValue: true, description: 'Enable build the release packages')
     booleanParam(name: 'its_ci', defaultValue: true, description: 'Enable async ITs')
+    choice(name: 'Build_type', choices: ['snapshot', 'staging'], description: 'Choose Snapshot or Staging Build type(Default: Snapshot)')
```

Comment:
seems like the casing convention should be `build_type` lowercase

Reply:
Also seems like the description specifies what the default value is, but there is no `defaultValue` argument.

Reply:
@amirbenun fixed casing convetion.
choice param doesn't support `defaultValue`, instead it takes the first param as default.